.circleci/config.yml

```diff
@@ -16,9 +18,8 @@ jobs:
           name: Install Dependencies
           command: |
             yarn
-            python3 -m venv venv
+            python3 -m venv venv 
             . venv/bin/activate
-            pip install pipenv
             pipenv install --dev
```

Comment:
Should we add `--ignore-pipfile` here to match the project Dockerfile?

Reply:
Yeah, I removed it while trying to debug.